src/org/oidc/service/AbstractService.java

```diff
@@ -0,0 +1,215 @@
+package org.oidc.service;
+
+import org.oidc.service.base.ServiceConfig;
```

Comment:
Could you please order the imports? Follow https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing
@lccodes FYI

Reply:
reorganized imports